### PR TITLE
Added additional cisco device types

### PR DESCRIPTION
**Pull request by abhi1693**

### CODE REVIEW

File: device-types/Cisco/WS-C3850-12X48U-S.yaml [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+manufacturer: Cisco
+model: Catalyst 3850-12X48U-S
+slug: cisco-ws-c3850-12x48u-s
+part_number: WS-C3850-12X48U-S
```
Comment (DanSheps) [inline device-types/Cisco/WS-C3850-12X48U-S.yaml:5]: So funny enough, the 3850 actually makes this distinction in inventory, even though the device itself is soft-upgradeable to a higher license level I believe.

```
NAME: "Switch 1", DESCR: "WS-C3850-48F-L"
PID: WS-C3850-48F-L    , VID: V07  , SN: <REDACTED>
```

File: device-types/Cisco/C9500-16X-A.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+manufacturer: Cisco
+model: Catalyst 9500-16X-A
+slug: cisco-c9500-16x-a
+part_number: C9500-16X-A
```
Comment (DanSheps) [inline device-types/Cisco/C9500-16X-A.yaml:5]: "-A" is just a license identifier for ordering.  The actual model number is C9500-16X.

AP's are different with the "-A" or "-I", etc as those are regional locks and the AP actually reports the specific model number for the region.

```
NAME: "Switch 1", DESCR: "C9500-16X"
PID: C9500-16X         , VID: V02  , SN:  <REDACTED>
```

**Review by DanSheps — CHANGES_REQUESTED** [review]
Might need weights.

The 9500-16X-A should only be a 9500-16X which might already be in the database.  The "-A" is just a ordering SKU

**Comment by abhi1693** [comment]
While I was adding the weights for the ones I submitted, I took the liberty to add the missing ones in the existing data as well.

File: device-types/Cisco/Meraki-MS225-24.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+manufacturer: Cisco
+model: Meraki MS225-24
+slug: cisco-meraki-ms225-24
+part_number: MS225-24-HW
+u_height: 1
+is_full_depth: false
+weight: 6.03
+weight_unit: lb
```
Comment (danner26) [inline device-types/Cisco/Meraki-MS225-24.yaml:9]: 2. Please add the `airflow` data point if you have it or can find it. This appears to be in the datasheet (passive)

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by lucatortiglione** [comment]
is there a way to fix the missing info by myself and close the pull ?

**Comment by danner26** [comment]
@lucatortiglione you can create a fork of this PR and then submit a new PR (and reference this one please). Otherwise no, unless he gives you access to his fork

**Comment by danner26** [comment]
@abhi1693 can you please let us know if you are going to work on this, and if not can you close it?

**Comment by danner26** [comment]
Closing this since it has been incomplete for a long period of time. Feel free to work on it then reopen it again if you would like.

**Pull request closed without merging**